std_msgs/Header header
float64 cam_std_y_dist_reader      # cam_std_y_dist_reader reading
float64 cam_std_y_dist_reader_var  # cam_std_y_dist_reader_var reading
